chapters/inheritance.tex

```diff
@@ -569,6 +569,16 @@ \subsection{Final Element Modification Prevention}\label{final-element-modificat
 end Test;
 \end{lstlisting}
 \end{example}
+If a class is final it means that instances of that class cannot have modifiers.
```

Comment:
What about extending from the class?
```
  model AnotherTransferFunction = MyTransferFunction;
  model YourTransferFunction = MyTransferFunction(a = {1, 2});
```

Reply:
Should be the same. Will reformulate.

Reply:
I still believe that the second example with `extends` could be worth including (showing that this is OK as long as there are no modifiers in the extends-clause).

Here's another situation that I think need clarification:
```
  model AnotherTransferFunction = MyTransferFunction;
  model YourTransferFunction = AnotherTransferFunction(a = {1, 2});
```

Reply:
I combined that with the extends-example.

Reply:
Conversation appears obsolete in view of new way of defining final class.  Resolving.